yt/yt/server/job_proxy/job_proxy.cpp

```diff
@@ -973,12 +973,12 @@ void TJobProxy::ReportResult(
         }
 
         try {
-            auto stderr = GetStderr();
-            if (!std::empty(stderr)) {
+            auto stderr = GetStderr({});
+            if (!stderr.Data.empty()) {
                 const auto& jobResultExt = result.GetExtension(TJobResultExt::job_result_ext);
                 YT_VERIFY(jobResultExt.has_stderr());
             }
-            req->set_job_stderr(std::move(stderr));
+            req->set_job_stderr(TString{stderr.Data.ToStringBuf()});
```

Comment:
I am not sure, but it seems that there is a method set with signature `(const char*, size_t)`, in this case I would prefer to use it and avoid extra allocation here.